swiftpkg/internal/clang_files.bzl

```diff
@@ -250,7 +250,7 @@ def _collect_files(
 
     modulemap = None
     modulemap_orig_path = None
-    for orig_path in all_srcs:
+    for orig_path in depset(all_srcs).to_list():
```

Comment:
Can you elaborate how we are getting duplicate sources in the first place?

Reply:
[This](https://github.com/facebook/yoga/pull/1779) needs to be merged first and then I'll update this PR

Reply:
Will you even need this PR, once the other PR merges?

Reply:
yes the other PR fixes something different related to picking up public headers unrelated to the compiled target